apis/nucleus/src/components/listbox/components/useTempKeyboard.js

```diff
@@ -18,19 +16,22 @@ export function getVizCell(container) {
 
 // Emulate the keyboard hook, until we support it in the Listbox.
 export default function useTempKeyboard({ containerRef, enabled }) {
-  const [keyboardActive, setKeyboardActive] = useState(false);
```

Comment:
removing the useState will no longer trigger a rerender when keyboardActive is changed
Is that a intentional?
and have you verified that it was not needed?
(I think it may be wanted for update of innerTabStops and tabIndex using it)